webapp/src/components/user_selector/autocomplete_selector.tsx

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import AsyncSelect from 'react-select/async';
+import {blendColors, changeOpacity, makeStyleFromTheme} from 'mattermost-redux/utils/theme_utils';
+
+import {Theme} from 'mattermost-redux/types/preferences';
+import {OptionsType, ValueType} from 'react-select/src/types';
+import {UserProfile} from 'mattermost-redux/types/users';
```

Comment:
Can we separate the external libraries (react-select) from pseudo-internal libraries (mattermost-redux)?

```suggestion
import AsyncSelect from 'react-select/async';
import {OptionsType, ValueType} from 'react-select/src/types';

import {blendColors, changeOpacity, makeStyleFromTheme} from 'mattermost-redux/utils/theme_utils';
import {Theme} from 'mattermost-redux/types/preferences';
import {UserProfile} from 'mattermost-redux/types/users';
```